os/drivers/lcd/Kconfig

```diff
@@ -145,22 +164,59 @@ config LCD_ST7785
 	---help---
 		Enable LCD Driver, ST7785 with mipi.
 
-if LCD_ST7785
-config LCD_ST7785_TYPEA
-	bool "ST7785 Type A LCD Driver"
+config LCD_ST7789
+	bool "ST7789 LCD Driver"
 	---help---
-		Enable LCD Driver, ST7785 with mipi.
+		Enable LCD Driver, ST7789 with spi.
+endchoice
 
-config LCD_ST7785_TYPEB
-	bool "ST7785 Type B LCD Driver"
+choice
+	prompt "LCD Initialization Mode"
+	default LCD_STATIC_INIT
 	---help---
-		Enable LCD Driver, ST7785 with mipi.
-endif
+		Select LCD initialization method
 
-config LCD_ST7789
-	bool "ST7789 LCD Driver"
+config LCD_DYNAMIC_INIT
+	bool "Dynamic initialization (read LCD ID)"
 	---help---
-		Enable LCD Driver, ST7789 with spi.
+		Initialize LCD by reading LCD ID and sending corresponding init sequence
+
+config LCD_STATIC_INIT
+	bool "Static initialization"
+	---help---
+		Initialize LCD using predefined init sequence
+
+if LCD_STATIC_INIT
+choice
+	prompt "Static Initialization code"
+	default LCD_STATIC_INIT_DEFAULT
+	---help---
+		Select predefined LCD initialization sequence
+
+config LCD_STATIC_INIT_DEFAULT
+	bool "Default initialization"
+	---help---
+		Use default initialization sequence
```

Comment:
What code is the default init code for which LCD? Is it necessary?

Reply:
prompt "Static Initialization code"
	default LCD_STATIC_INIT_AVD_0027NNB01
	
	you can select default setting using above code